CONTRIBUTING.md

```diff
@@ -139,6 +143,83 @@ The description field should
+* Encouragement to provide example FMUs and any information needed to run this FMU and check expected results
+  * check with one of the available checkers before uploading the FMUs (fmpy, VDM Checker, FMU Compliance Checker for FMI 2.0), --> self-certification
+  * description, including with which version of the tool the FMU was generated
+  * (copy description of files from https://github.com/modelica/fmi-cross-check/blob/master/FMI-CROSS-CHECK-RULES.md#914-files-to-submit: TODO: Define if reference result refers to result of the simulation of the original model in the exporting tool, or to the simulation of the FMU)
+  * recommendation: structure your repository according to FMI versions, FMI interface types (ME, CS, SE), platforms (consider to provide source code FMUs)
+
+
+Tool-specific-repository.org:
+```
+2.0
+..
+3.0
+  ME
+    x86_64-windows
+      BouncingBall
+      CoupledClutches
+        CoupledClutches.fmu 
+        CoupledClutches_in.csv 
+        CoupledClutches_ref.csv 
+        CoupledClutches_ref.opt
+        Readme.txt
+    x86-linux  
+    ..
+    source-code
+  CS
+  SE
+  ```
+
+#### Importing tools
+
+* List exporting tools (including version information), from which FMUs have been successfully imported and simulated.
+  * recommended: list these FMUs and/or provide links to the repository of the exporting tool
+  * optionally additional information needed to reproduce the results of the importers (e.g., compiler flags)
+
+#### CSV Rules for inout and result files for compatibility information
+
+CSV as input, output and reference file format is convenient as many tools support CSV import and export.
+However, CSV is not restrictive enough to allow easy exchange of time-based signals.
+Here we will declare a number of additional restrictions on top of the CSV format to ease handling:
+
+- Separator: comma (`,`): separators may only be used between elements, not the end of a line
+
+- Numbers must be unquoted and specified in the format used for floating point literals as in the C programming language (without the type suffix).
+
+- All numeric cell entries contain numbers, labels for enumerations are not allowed (it would require readers to have access to the FMU information).
+Boolean values should be expressed as `0` and `1`.
+Binary values and strings are represented as for start values in the modeldescription.xml
+Arrays are serialized as for start values in the modelDescription.xml with space (` `) as separator.
+
+- The first column contains the time.
+The values in time must be monotonically increasing.
+
+- The first line contains the variable names quoted with double quotes (`"`)
+
+- All variable names are exactly the same as defined in the modelDescription.xml (no additional variables are allowed)
+
+- Starting with line 2, data is supplied (no units in line 2, no comments allowed)
+
+- A CSV file must not be larger than 1 Megabyte.
```

Comment:
Should we still care about this size? We are now longer responsible for hosting it. I would at least change it to a recommendation instead of "MUST NOT"